Containerfile.image_build

```diff
@@ -25,20 +25,25 @@ RUN make buildah
 # that runs safely with privileges within the container.
 #
 
-FROM registry.fedoraproject.org/fedora-minimal:40
+FROM quay.io/centos/centos:stream10-minimal
 
 LABEL "io.containers.capabilities"="CHOWN,DAC_OVERRIDE,FOWNER,FSETID,KILL,NET_BIND_SERVICE,SETFCAP,SETGID,SETPCAP,SETUID,CHOWN,DAC_OVERRIDE,FOWNER,FSETID,KILL,NET_BIND_SERVICE,SETFCAP,SETGID,SETPCAP,SETUID,SYS_CHROOT"
-LABEL name="buildah" \
-      summary="Containerized version of buildah which can be used to build OCI artifacts" \
-      com.redhat.component="konflux-ci-buildah" \
+LABEL \
+      org.label-schema.schema-version="1.0" \
+      org.label-schema.vendor="Konflux CI" \
+      org.opencontainers.image.vendor="Konflux CI" \
+      org.opencontainers.image.url="https://quay.io/konflux-ci/buildah" \
+      org.opencontainers.image.source="https://github.com/konflux-ci/buildah-container" \
+      org.label-schema.name="buildah" \
+      org.opencontainers.image.title="buildah" \
+      name="konflux-buildah" \
+      com.redhat.component="konflux-buildah" \
+      io.k8s.display-name="konflux-buildah" \
+      io.openshift.tags="buildah oci" \
+      summary="Containerized version of buildah which can be used to build OCI artifacts within Konflux CI." \
       description="OCI images and artifacts are central to the architecture of Konflux. In order to ensure that we can always take advantage of the latest functionality, this image can be used to reliably build the latest version of the CLI." \
-      io.k8s.display-name="buildah" \
       io.k8s.description="OCI images and artifacts are central to the architecture of Konflux. In order to ensure that we can always take advantage of the latest functionality, this image can be used to reliably build the latest version of the CLI." \
-      io.openshift.tags="oci" \
-      vendor="Red Hat, Inc." \
-      distribution-scope="public" \
-      release="0" \
-      url="github.com/konflux-ci/buildah-container"
+      org.opencontainers.image.documentation="https://github.com/containers/image_build/blob/main/buildah/README.md"
```

Comment:
It would be good to use OCI annotations consistently.

Reply:
The CenOS Stream images use `org.label-schema` so I wanted to overwrite those. I don't think that we currently have the ability to set annotations on our builds. It would be handy if our buildah tasks would copy all labels to annotations though as that seems like it is a preferred pattern.